server/pkg/mail/healthcheck.go

```diff
@@ -17,3 +19,32 @@ func Healthcheck(ctx context.Context) error {
 
 	return nil
 }
+
+func SMTPConfigurationCheck(host string, port uint, username string, password string, tlsConfig *tls.Config) (expected string) {
```

Comment:
1) this is not a healthcheck, move this to `check.go`
2) remove the smtp prefix - just ConfigCheck 
3) `expected` sounds a bit off, do we have a better alternative?